### PR TITLE
Replace all instances of plot.ly with plotly.com.

### PR DESCRIPTION
**Pull request by shammamah**
### About
Related to https://github.com/plotly/dash-core/issues/152

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -736,7 +736,7 @@ Big thanks to community contributor @Akronix!
 
 ## [0.14.0] - 2017-10-17
 ### :sparkles: Added
-- An `Upload` component! :tada: See [https://plot.ly/dash/dash-core-components/upload](https://plot.ly/dash/dash-core-components/upload) for docs.
+- An `Upload` component! :tada: See [https://plotly.com/dash/dash-core-components/upload](https://plotly.com/dash/dash-core-components/upload) for docs.
```
Comment (shammamah) [inline CHANGELOG.md]: Should I change this to dash.plotly.com/dash/... instead?

Comment (Marc-Andre-Rivet) [reply CHANGELOG.md]: Yes. Or rather `dash.plotly.com/dash-core-components`

**Comment by Marc-Andre-Rivet** [comment]
@shammamah Needs update. Test failures look like noise.

File: CHANGELOG.md (hunk 1 above)
Comment (Marc-Andre-Rivet) [reply CHANGELOG.md]: No. Redirects are handling these just fine.

**Pull request merged**